packages/theming/hash.txt

```diff
@@ -1 +1 @@
-CQhk8M6oN0XzkZBkgImthR2kdTs=
\ No newline at end of file
+rZnyMTmoc2CFM82C+7zGHcw3GGY=
```

Comment:
Maybe the hash generated from your local build should be excluded. I'm not sure if this is an issue.

Reply:
Nort quite sure. I've seen it being committed with other changes